projects/1.20/assets/slashblade/slashblade/lang/zh_cn.json

```
@@ -0,0 +1,179 @@
+  "slashblade.tooltip.material.broken": "断刀",
+  "slashblade.tooltip.material.noscabbard": "无鞘",
+  "slashblade.tooltip.material.enchantments": "附魔",
+  
+  "item.slashblade.slashblade": "无铭「无名」",
+  "item.slashblade.slashblade.desc": "合成修理的悲剧\n无垢之器",
+  "item.slashblade.simple.wood": "无铭刀「木偶」",
+  "item.slashblade.simple.wood.desc": "带鞘的木刀",
+  "item.slashblade.simple.white": "利刀「白鞘」",
+  "item.slashblade.simple.white.desc": "铁制的刀刃",
+  "item.slashblade.simple.white.material": "刀组件「白」",
+  "item.slashblade.simple.bamboo": "无铭刀「竹光」",
+  "item.slashblade.simple.bamboo.desc": "未完成的工艺品",
+  "item.slashblade.simple.bamboo.material": "刀组件「竹」",
+  "item.slashblade.simple.silverbamboo": "名刀「银纸竹光」",
+  "item.slashblade.simple.silverbamboo.desc": "工艺品",
+  "item.slashblade.simple.silverbamboo.material": "刀组件「虚荣」",
+  "item.slashblade.simple.iron": "大太刀「」",
+  "item.slashblade.simple.iron.desc": "伊始",
+  "item.slashblade.simple.iron.material": "刀组件「铁」",
+
+  "item.slashblade.ex.ruby": "利刀「无名」红玉",
+  "item.slashblade.ex.ruby.desc": "杰作",
+  "item.slashblade.ex.ruby.material": "刀组件「红玉」",
+
+  "item.slashblade.ex.fox.white": "狐月刀「白狐」",
+  "item.slashblade.ex.fox.white.desc": "幸运的标志",
+  "item.slashblade.ex.fox.white.material": "刀组件「白狐」",
+  "item.slashblade.ex.fox.black": "狐月刀「黑狐」",
+  "item.slashblade.ex.fox.black.desc": "安宁的标志",
+  "item.slashblade.ex.fox.black.material": "刀组件「黑狐」",
+
+  "item.slashblade.ex.tukumo": "宝刀「付丧」结月",
+  "item.slashblade.ex.tukumo.desc": "结月刀「付丧」\n铁制刀刃的轨迹",
+  "item.slashblade.ex.tukumo.material": "刀组件「付丧」",
+
+  "item.slashblade.ex.muramasa": "「千鹤」村正",
+  "item.slashblade.ex.muramasa.desc": "失传的名刀村正",
+  "item.slashblade.ex.muramasa.material": "刀组件「村正」",
+
+  
+  "item.slashblade.proudsoul": "耀魂碎片",
+  "item.slashblade.proudsoul_tiny": "破碎的耀魂",
+  "item.slashblade.proudsoul_ingot": "耀魂铁锭",
+  "item.slashblade.proudsoul_sphere": "耀魂宝珠",
+  "item.slashblade.proudsoul_crystal": "耀魂结晶",
+  "item.slashblade.proudsoul_trapezohedron": "耀偏方三八面魂",
+  "item.slashblade.proudsoul_activated":"饥渴的耀偏方三八面魂",
+  "item.slashblade.proudsoul_awakened":"玄奥的耀偏方三八面魂",
+  "item.slashblade.proudsoul_tiny.dark":"破碎的耀魂之茧",
+
+  "entity.slashblade.blade_stand_entity": "刀挂台",
+  "item.slashblade.bladestand_1": "立式单刀挂台",
+  "item.slashblade.bladestand_2": "立式双刀挂台",
+  "item.slashblade.bladestand_v": "立式竖刀挂台",
+  "item.slashblade.bladestand_s": "立式短刀挂台",
+  "item.slashblade.bladestand_1w": "挂式单刀挂台",
+  "item.slashblade.bladestand_2w": "挂式双刀挂台",
+
+  "adv.slashblade.proudsoul": "强化限制=50\n利刃易折\n断刃之中高傲的灵魂之火",
+  "adv.slashblade.proudsoul_tiny": "强化限制=10\n幽微闪烁的灵魂之炎",
+  "adv.slashblade.proudsoul_ingot": "强化限制=100\n以高傲紫色之炎锻造的灵铁",
+  "adv.slashblade.proudsoul_sphere": "强化限制=150\n熔炉冶炼\n缭绕着虚幻灵魂之炎的宝珠",
+  "adv.slashblade.proudsoul_crystal": "强化限制=200\n高炉精炼\n灵魂钢铁的核心",
+  "adv.slashblade.proudsoul_trapezohedron": "无限制强化\n烟熏炉熏烤\n摇曳的高傲之结晶\n此乃千锤百炼之终末",
+  "adv.slashblade.proudsoul_activated":"唤醒它的代价是无尽的饥饿\n潜行并单击右键使用以激活耀偏方三八面魂\n放在快捷栏时会被饥饿所侵袭\n饥饿状态下将继续以饥饿修复刀剑",

```

```suggestion
  "adv.slashblade.proudsoul_activated":"唤醒它的代价是无尽的饥饿\n潜行并右击使用以激活耀偏方三八面魂\n放在快捷栏时会被饥饿所侵袭\n饥饿状态下将继续以饥饿修复刀剑",
```
啊还有个这个小问题
保不保留`使用`（右击以激活 or 右击使用以激活）看你了

---

前者吧，简短不冗余